nautobot/ipam/tests/test_api.py

```diff
@@ -506,7 +506,7 @@ def test_create_single_available_prefix(self):
             url, {"prefix_length": "hello", "status": self.status.pk}, format="json", **self.header
         )
         self.assertHttpStatus(response, status.HTTP_400_BAD_REQUEST)
-        self.assertIn("prefix_length", response.data[0])
+        self.assertIn("prefix", response.data)
```

Comment:
Change here seems wrong to me - shouldn't our 400 response still indicate that `prefix_length` was the invalid input parameter?

Reply:
I think it is because `request_prefixes` no longer contains the "prefix" field because we are not using serializer.validated_data. Let me see what I can come up with.